src/dmd/astbase.d

```diff
@@ -3995,13 +4007,165 @@ struct ASTBase
             t.isproperty = isproperty;
             t.isref = isref;
             t.isreturn = isreturn;
-            t.isscope = isscope;
-            t.iswild = iswild;
+            t.isthisscope = isthisscope;
+            t.isreturninferred = isreturninferred;
+            t.isscopeinferred = isscopeinferred;
+            t.isInOutParam = isInOutParam;
+            t.isInOutQual = isInOutQual;
             t.trust = trust;
             t.fargs = fargs;
             return t;
         }
 
+        /// set or get if the function has the `nothrow` attribute
+        bool isnothrow() const pure nothrow @safe @nogc
```

Comment:
Won't accessor functions eat up the saved space if you don't mark them `final`, or do `extern(C++)` objects not carry around function pointers the way `extern(D)` objects do?

Reply:
they are final because `TypeFunction` is `final class`